swagger_client/__init__.py

```diff
@@ -21,16 +21,27 @@
 from swagger_client.configuration import Configuration
 # import models into sdk package
 from swagger_client.models.action import Action
+from swagger_client.models.action_type import ActionType
```

Comment:
How were these changes not already in the `development` tree?  Did we forget something in the previous ticket?

Reply:
That is an excellent question that I do not know the answer to!  We must have done something wrong as they should be in there.